pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -224,6 +224,7 @@ Servers
       tcpSendTimeout=NUM,    -- The timeout (in seconds) of a TCP write attempt
       tcpRecvTimeout=NUM,    -- The timeout (in seconds) of a TCP read attempt
       tcpFastOpen=BOOL,      -- Whether to enable TCP Fast Open
+      ipBindAddrNoPort=BOOL, -- Whether to enable IP Bind Address No Port
```

Comment:
"Whether to enable IP_BIND_ADDRESS_NO_PORT if available, default: true"?

Reply:
Thanks @rgacogne. Updated with that documentation change.